src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -8200,16 +8197,37 @@ public long xlen(final String key) {
     return connection.executeCommand(commandObjects.xrevrange(key, end, start));
   }
 
-  /**
-   * {@inheritDoc}
-   */
   @Override
   public List<StreamEntry> xrevrange(final String key, final StreamEntryID end,
```

Comment:
According to the [XRANGE docs](https://redis.io/commands/XRANGE) this should be start, end, count instead of end, start, count.

This applies to all signatures.

Reply:
@chayim Please check, XRANGE is fine.

This is [XREVRANGE](https://redis.io/commands/xrevrange). So that's fine as well.

Reply:
DERP You are right. Eye blead.